spread.yaml

```diff
@@ -16,17 +16,31 @@ backends:
                 kernel: GRUB 2
             - debian-unstable-64:
                 kernel: GRUB 2
+            - fedora-25-64
 
 path: /root/spread
 
 restore: |
-    apt autoremove -y
+    case "$SPREAD_SYSTEM" in
+        fedora-*)
+            dnf clean all -y
+            dnf clean metadata
```

Comment:
In my tests I had to do a `dnf clean metadata` afterwards as there were still things lying around.